pkg/operator/handler.go

```diff
@@ -315,6 +315,10 @@ func (h *Handler) Handle(ctx context.Context, event sdk.Event) error {
 			if errOp != nil {
 				logrus.Errorf("unable to update cluster status to %s=%s: %s", osapi.OperatorAvailable, osapi.ConditionTrue, errOp)
 			}
+			errOp = h.operatorStatusVersion(cr.Spec.Version)
+			if errOp != nil {
+				logrus.Errorf("unable to update cluster status version to %s: %s", cr.Spec.Version, errOp)
+			}
```

Comment:
this seems like a strange place to be setting this value from.

Fundamentally we should just be asserting ClusterOperator's version any time we process a cr event.  (obviously if it hasn't changed, no update is needed).

Reply:
@bparees I added this because of this paragraph (from the https://jira.coreos.com/browse/NE-92):
```
The operator must set Version to its version string (e.g., "v1.2.3"). The update payload will specify a manifest for the operator. CVO sees the version specified by the update payload and will wait until the operator reports the same version.
```
My understanding of this is that we must expose the version of the registry resource that we were asked to do. Operator version does not change. The version of the image-registry operator does not change as we do not update ourselves. Therefore, this cannot be a version of the operator itself.

I indicate the custom resource version when I succeeded. Up to this point, the previous version is working.

Reply:
My understanding that it should be the version of our operator so that CVO can know that the operator's upgrade was successful. But the thing that I don't understand is how CVO will know which version it should expect to see.

Reply:
not sure either. @jwforres any idea where the CVO gets our operators version from?  Or where our operator is supposed to get its version from?

I don't know if that level of status checking is wired up yet.

@legionus for now just hard code it something (`v0.0.1`) and leave a TODO for us to come back to it.  My primary concern for demos on wednesday is that we are creating the ClusterOperator object and setting some basic values in it.  As I said in my email and this morning, getting to the final implementation specifics will be an iterative process.

Reply:
I believe you want to put your version onto your ClusterOperator's status.version, on my 4.0 cluster from yesterday there are several ClusterOperator instances being created now and they all report this there

cc @abhinavdahiya

Reply:
yeah this came up at lunch and basically:

1) the version value is entirely up to us.  at some point i'd like it to be based in part on the git commit, but for now, just hardcode it and let's move on.

2) the CVO (or whatever is consuming the ClusterOperator values) has no way to know what the value is supposed to be.  Whether it ever will is unclear, but again, for now, just set it to something and move on.